### ISSUE TITLE
Can't resolve 'url' with webpack v5

### ISSUE THREAD

**Issue by johnmanko** [mat-sz/upload#4]
```

ERROR in ./node_modules/upload/lib/Upload.js 18:12-26
Module not found: Error: Can't resolve 'url' in '.../node_modules/upload/lib'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "url": require.resolve("url/") }'
        - install 'url'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "url": false }
resolve 'url' in '.../node_modules/upload/lib'
  try alternate dist: /dist/native-modules/ in only request
    Parsed request is a module
    using description file: .../node_modules/upload/package.json (relative path: ./lib)
```

**Comment by johnmanko** [comment]
Consider using the native browser URL: https://developer.mozilla.org/en-US/docs/Web/API/URL

Otherwise, which polyfill do you recommend?  A `peerDependencies` package should be specified if the native URL class isn't used.

**Comment by mat-sz** [comment]
Fixed in v1.2.0
